code/obj/machinery/computer/card.dm

```diff
@@ -38,6 +38,14 @@
 	icon = 'icons/obj/computerpanel.dmi'
 	icon_state = "id2"
 
+/obj/machinery/computer/card/portable
+  name = "portable identification computer"
+  icon_state = "idportC"
+
+  New()
+    ..()
+    src.AddComponent(/datum/component/foldable,/obj/item/objBriefcase/blue_green_stripe)
```

Comment:
Indent here is wrong. Use tabs rather than spaces.

Reply:
fixed, thanks